eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -234,7 +234,6 @@ private function internalPublishUrlAliasForLocation(
             // 2. existing location or custom alias entry
             // 3. history entry
```

Comment:
Not sure about the initial intention behind this, but this comment should be updated too concerning your changes.